Feature2/us 048 migrate post operation to login

Sergio-Dom opened the pull request.

Sergio-DC committed:
Orika dependecy added, response fields configured

- Trace field disabled, was necessary to remove spring-boot-devtools
- Message field enabled, application.propeties configured

Sergio-DC committed:
Validators & Exceptions migrated

Sergio-DC committed:
Services migrated (Only what is necessary)

Sergio-DC committed:
DTOs migrated

Sergio-DC committed:
Repository class renamed & new method added

Sergio-DC committed:
New field added to User model

Sergio-DC committed:
Mapper & Controller migrated

Sergio-DC committed:
Repository class referenced changed

'va' file ignored

Sergio-DC committed:
Import names fixed

Sergio-DC committed:
Modifying validators & exceptions to match sce-api repo

Sergio-DC committed:
Apache commons dependency added

Sergio-DC committed:
GlobalExceptionBody migrated

Sergio-DC committed:
Code from at-sce-api to at-sso-api integrated

Sergio-DC committed:
Unit tests added (Only for login functionality)

Sergio-DC committed:
Expected output of test cases modified

Sergio-DC committed:
Exclusion tags removed to enable execution of JUnit

luisgalindoluna reviewed (approved):
I see you've finally figured this out, well done

EmiliaGuillen-dev reviewed (approved):
looks good

Sergio-DC committed:
Merge branch 'develop' into feature2/US_048_Migrate_Post_operation_to_login

Sergio-DC committed:
2 new classMaps added2 new classMaps added

Sergio-DC committed:
Merge branch 'develop' into feature2/US_048_Migrate_Post_operation_to_login

Sergio-DC committed:
Repository File restored

Sergio-DC committed:
Login user method added with his validator

Sergio-DC committed:
Login controller method added

Sergio-DC committed:
Exception thrown by LoginValidator fixed

- @Autowired set to LoginValidator Instance
- Import reference to Validation Utils changed

Sergio-DC committed:
New login test cases added
